Sources/AppCoinsSDK/Models/Services/Gamification/AppCoinGamificationServiceClient.swift

```diff
@@ -15,9 +15,9 @@ internal class AppCoinGamificationServiceClient : AppCoinGamificationService {
         self.endpoint = endpoint
     }
     
-    internal func getTransactionBonus(address: String, package_name: String, amount: String, currency: Coin, result: @escaping (Result<TransactionBonusRaw, TransactionError>) -> Void) {
+    internal func getTransactionBonus(address: String, package_name: String, amount: String, currency: String, result: @escaping (Result<TransactionBonusRaw, TransactionError>) -> Void) {
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done